src/components/FetchApi.js

```diff
@@ -59,12 +64,12 @@ export default class FetchApi extends Component {
         })
     }
 
-    setAddPost=(addPostTitle,addPostBody)=>{
-        this.setState({
-            title:addPostTitle,
-            body:addPostBody
-        })
-    }
+   
+
+submitHandler=(e)=>{
+    e.preventDefault()
+    console.log(this.state.title,this.state.body)
+}
 
   //Create a functional component for rendering users 
     renderUser = () => {
```

Comment:
in this function, you have used the map array function, check out the importance of using a key in react for child elements